src/js/components/tasks/cards/index.tsx

```diff
@@ -58,6 +60,9 @@ const OrgCards = ({
   const [isDeletingOrg, setIsDeletingOrg] = useState<OrgTypeTracker>(
     OrgTypeTrackerDefault,
   );
+  const [isRefreshingOrg, setIsRefreshingOrg] = useState<OrgTypeTracker>(
+    OrgTypeTrackerDefault,
+  );
```

Comment:
I don't think we need to track this in local state, since we have a boolean flag on the org object itself.